tests/View/ViewTest.php

```diff
@@ -203,6 +203,14 @@ public function testWithErrors()
 	}
 
 
+    public function testViewExceptionHandling()
```

Comment:
No offense - but will worry about that once I know what sort of solution I should be going with :)